user/db.go

```diff
@@ -21,29 +20,27 @@ func (s *Store) countUsers(ctx context.Context) (int, error) {
 }
 
 // countUsers24Hours will get the number of users in the last 24 hours
-// TODO make this a lot less bad, it just is bad
 func (s *Store) countUsers24Hours(ctx context.Context) (int, error) {
 	count := 0
 	err := s.db.GetContext(ctx, &count,
 		`SELECT COUNT(*)
 		FROM people.users
-		WHERE last_login > timestamp '`+time.Now().AddDate(0, 0, -1).Format("2006-01-02 15:04:05")+`';`)
+		WHERE last_login > TO_TIMESTAMP($1, 'YYYY-MM-DD HH24:MI:SS');`, time.Now().AddDate(0, 0, -1).Format("2006-01-02 15:04:05"))
```

Comment:
More for curiosity than an issue - why was this change necessary?

Reply:
That was part of a TODO and put it through sql parsing rather than inline concat